opta/core/aws.py

```diff
@@ -435,17 +435,22 @@ def cluster_exist(self) -> bool:
         except client.exceptions.ResourceNotFoundException:
             return False
 
+    def get_kubectl_context_name(self) -> str:
+        region, account_id = self.get_cluster_env()
```

Comment:
you can have a region and account without a cluster - may be we we should also rename it `get_provider_data` or `get_cloud_env` or something that is not tied to a cluster.

I realized this is not a new method - but since we use it, we can rename it if we wish.

Reply:
nah, let's keep it as is for now